Simplenote/src/androidTest/java/com/automattic/simplenote/utils/TestBucket.kt

```diff
@@ -0,0 +1,52 @@
+package com.automattic.simplenote.utils
+
+import com.simperium.client.Bucket
+import com.simperium.client.BucketObjectMissingException
+import com.simperium.client.Syncable
+
+abstract class TestBucket<T : Syncable>(name: String) : Bucket<T>(
```

Comment:
1) Praise (❤️): This is nicely thought, kudos!
2) Minor (🔍): Consider reformatting this class. I believe the formatting for this class can be further improved (see constructor).

Reply:
Is there a specific standard for Kotlin conventions that you follow at Automattic? I have changed the constructor but I am not sure if it complies with your standard.

Reply:
👋 @dandominguez04 !

> Is there a specific standard for Kotlin conventions that you follow at Automattic? I have changed the constructor but I am not sure if it complies with your standard.

We are trying to use the default Kotlin style guide, the official one. And since this repo is just starting on Kotlin, I suggest we do the exact same thing, just use the default Kotlin style guide, the exact same thing that the community is using.

So, in this occasion, the only change would be on the constructor indeed.

Reply:
PS: I was think of reformatting that and keeping the multiline approach that you started with (see below), but the one line approach you ended up with is ok too.

```
abstract class TestBucket<T : Syncable>(name: String) : Bucket<T>(
    null,
    name,
    null,
    null,
    null,
    null
) {
```